packages/next/src/shared/lib/router/router.ts

```diff
@@ -520,6 +524,7 @@ function fetchNextData({
         isPrefetch && hasMiddleware ? { 'x-middleware-prefetch': '1' } : {}
       ),
       method: params?.method ?? 'GET',
+      priority
```

Comment:
i would just set it here
```suggestion
      priority: isPrefetch ? 'low' : 'auto'
```